src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts

```diff
@@ -269,6 +276,18 @@ class ZWaveJSConfigDashboard extends LitElement {
     );
   }
 
+  private async _openOptionFlow() {
+    const searchParams = new URLSearchParams(window.location.search);
+    if (!searchParams.has("config_entry")) {
```

Comment:
`this.configEntryId` should be passed in from the router so we shouldn't need to pull it from the URL search params here